### PR TITLE
Repr for skills

### PR DESCRIPTION
**Pull request by oddlyspookycherry**
## Description
Added __repr__() debugging method to all skills. After review delete the calls to __repr__() from role/passer.

## Steps to test
Run the simulation.

Expected result: console output.

### CODE REVIEW

File: rj_gameplay/rj_gameplay/role/passer.py [hunk 1]
```diff
@@ -41,6 +41,10 @@ def tick(self, world_state: stp.rc.WorldState) -> RobotIntent:
             self._state = "receiving"
         elif self._state == "receiving":
             intent = self.receive_skill.tick(world_state)
+
+            print("ReceiveSkill:***************************************************")
+            print(self.receive_skill)
+
```
Comment (oddlyspookycherry) [inline rj_gameplay/rj_gameplay/role/passer.py]: Remove after review.

File: rj_gameplay/rj_gameplay/role/passer.py [hunk 2]
```diff
@@ -58,6 +62,9 @@ def tick(self, world_state: stp.rc.WorldState) -> RobotIntent:
             self._state = "execute_pass"
         elif self._state == "execute_pass":
             intent = self.pivot_kick_skill.tick(world_state)
+            
+            print("PivotKickSkill:***************************************************")
+            print(self.pivot_kick_skill)
```
Comment (oddlyspookycherry) [inline rj_gameplay/rj_gameplay/role/passer.py]: Remove after review.

File: rj_gameplay/rj_gameplay/gameplay_node.py [hunk 3]
```diff
@@ -456,8 +456,7 @@ def shutdown(self) -> None:
 
 def main():
     play_selector = basic_play_selector.BasicPlaySelector()
-
-    # change this line to test different plays (set to None if no desired test play)
+    # play_selector = None
```
Comment (kfu02) [inline rj_gameplay/rj_gameplay/gameplay_node.py]: should put back the comment that was here originally (can simply `git checkout` the file from the base branch)

**Review by kfu02 — CHANGES_REQUESTED** [review]
Looks good, just one small thing to clean up before merge!

**Review by kasohrab — APPROVED** [review]
looks good to me

**Pull request merged**